shopfloor/actions/search.py

```diff
@@ -168,12 +168,18 @@ def generic_packaging_from_scan(self, barcode):
             [("barcode", "=", barcode), ("product_id", "=", False)], limit=1
         )
 
+    def _get_origin_from_barcode(self, barcode):
+        """Returns a list of valid origins for a given barcode."""
+        # Hookable method
+        return [barcode]
+
     def origin_move_from_scan(self, barcode, extra_domain=None):
```

Comment:
I'm not sure to understand the goal of this method, a docstring is welcome.
From what I see, in `shopfloor_reception` we are overloading this method, are we not breaking something here for scenarios using this method not in a return/reception context?

Reply:
ping

Reply:
any answer to this?